packages/esm-ward-app/src/hooks/useRestPatient.ts

```diff
@@ -0,0 +1,18 @@
+import { openmrsFetch, restBaseUrl, type FetchResponse, type Patient } from '@openmrs/esm-framework';
+import { useMemo } from 'react';
+import useSWRImmutable from 'swr/immutable';
+
+export default function useRestPatient(patientUuid: string) {
```

Comment:
I would just call this usePatient... and I would figure a hook like this would already exist in esm-core, but if not fine to add it here, at least for now.

Reply:
Actually, we already have a hook `usePatient` in the framework, which fetches the FHIR Patient, so not to have conflicting names, I used `useRestPatient`.

Reply:
Ah, okay, makes sense!